.github/workflows/promote-ga.yml

```diff
@@ -2,7 +2,7 @@ name: promote-to-ga
 on:
   push:
     tags:
-      - v*
+      - 'v2.[0-9]+.[0-9]+'
```

Comment:
not sure how I feel about hardcoding v2 here. Can we just add another integer clause?

Reply:
What I'm particularly worried about is a chart version (`v7.*`) ending up here.

Reply:
Changed.